library/src/scripts/components/mebox/pieces/VanillaHeaderNav.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * @author Stéphane LaFlèche <stephane.l@vanillaforums.com>
+ * @copyright 2009-2018 Vanilla Forums Inc.
+ * @license GPL-2.0-only
+ */
+
+import * as React from "react";
+import classNames from "classnames";
+import VanillaHeaderNavItem, { IVanillaHeaderNavItemProps } from "./VanillaHeaderNavItem";
+
+export interface IVanillaHeaderNavProps {
+    className?: string;
+    children: IVanillaHeaderNavItemProps[];
```

Comment:
Same deal with naming children. `navItems` maybe?

Reply:
I want to differentiate them from the panel nav, but opted for a shorter: "IHeaderNav"